pandas/tests/io/parser/test_read_fwf.py

```
@@ -699,15 +699,15 @@ def test_encoding_mmap(memory_map):
     GH 23254.
     """
     encoding = "iso8859_1"
-    data = BytesIO(" 1 A Ä 2\n".encode(encoding))
```

This test wasn't using memory_map because it silently failed.